packages/dashboard/server/getElement.ts

```diff
@@ -0,0 +1,41 @@
+import { Element } from '@sidetree/element';
+import { EthereumLedger } from '@sidetree/ethereum';
+import Web3 from 'web3';
+import { IpfsCasWithCache } from '@sidetree/cas-ipfs';
+import config from '../config/test.json';
+
+const getTestLedger = async () => {
+  const web3 = new Web3(config.ethereumRpcUrl);
+  const ledger = new EthereumLedger(web3, config.elementAnchorContract);
+  return ledger;
+};
+
+const getTestCas = async () => {
+  const cas = new IpfsCasWithCache(
+    config.contentAddressableStoreServiceUri,
+    config.mongoDbConnectionString + config.databaseName
+  );
+  return cas;
+};
+
+async function createElement() {
+  const ledger: any = await getTestLedger();
+  const cas: any = await getTestCas();
+  const element = new Element(config as any, config.versions, ledger, cas);
+  await element.initialize(
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    { info: () => {}, warn: () => {}, error: () => {} },
```

Comment:
XD

Reply:
Still didn't work :/ I'm getting a ton of noise in the logs